junebug/stores.py

```diff
@@ -0,0 +1,64 @@
+from twisted.internet.defer import inlineCallbacks, returnValue
+from twisted.web import http
+from vumi.message import TransportUserMessage
+
+from junebug.error import JunebugError
+
+
+class MessageNotFound(JunebugError):
+    '''Raised when a message cannot be found.'''
+    name = 'MessageNotFound'
+    description = 'message not found'
+    code = http.NOT_FOUND
+
+
+class BaseStore(object):
+    '''Base class for store objects. Stores data in redis in a hash.
+    redis: redis manager
+    ttl: expiry for the key
```

Comment:
Does this docstring need to be updated to suggest that the ttl is for all keys managed by this store?